style.css

```
@@ -39,3 +81,50 @@ p {
   box-shadow: 0 0 25px rgba(0, 0, 0, 0.25);
   background: white;
 }
+
+.inputNewTasks{
+  height: 50px;
+  padding: 8px;
+  flex-direction: row;
+  align-items: stretch;
+  display:flex;
+  font-family: 'NEXON Lv1 Gothic OTF';
+  margin-top: auto;
+  box-shadow: 0 0 25px rgba(0, 0, 0, 0.25);
+
+}
+
+
+
+#todoInput{
+  flex: 1; /*specify it for a child if you wnat it to fill the remaining horizontal spaces*/
+  border-radius: 10px;
+  padding: 0.2px;
+  margin-right: 5px;
+  text-align: center;
+  font-size: 1em;
+
+}
+
+#addButton{
+  width : 33px;
+  height : 33px;
+  margin-right: 3px;
+  border-radius: 20px;
+
+  text-align: center;
+
+  background-color: #2ec1ac;
+  -webkit-text-fill-color: white;
```

`-webkit-text-fill-color` 는 표준문법도 아니고, 불필요한 문법이랍니당

```suggestion
  color: white;
```